### PR TITLE
Default unconfigured monitors to open to the right

### PR DESCRIPTION
**Pull request by eriedaberrie**
#### Describe your PR, what does it fix/add?
This PR returns the functionality of making unconfigured monitors default to opening to the right (previously all the if statements fell through and they ended up opening up, the enum zero value). It also improves the logging for invalid auto directions. Finally, it makes the logging performed in arrangeMonitors actually accurate under the new directional auto system, instead of just stating the next available rightwards position even when the new monitor isn't being arranged to the right.

I also noticed that since the max monitor positions are being stored as ints, there was inconsistent use of decimals in the arrangeMonitor logging. Seeing as how monitor positions can only be integers values anyways, I opted to just remove all decimals in the logging.

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)
I'm not too sure if the logging changes are really in the scope of the PR but the inconsistencies were really bugging me and I didn't want to make a whole new PR to edit logging.

### CODE REVIEW

**Review by vaxerski — APPROVED** [review]
lgtm

**Comment by vaxerski** [comment]
ah clang-format needed

**Comment by eriedaberrie** [comment]
Whoops I accidentally pushed to the wrong branch just now.
(clang-format has been performed)

File: src/config/ConfigManager.cpp [hunk 1]
```diff
@@ -1647,15 +1651,20 @@ std::optional<std::string> CConfigManager::handleMonitor(const std::string& comm
     if (ARGS[2].starts_with("auto")) {
         newrule.offset = Vector2D(-INT32_MAX, -INT32_MAX);
         // If this is the first monitor rule needs to be on the right.
-        if (ARGS[2] == "auto-right" || ARGS[2] == "auto" || m_dMonitorRules.empty())
+        if (ARGS[2] == "auto-right" || ARGS[2] == "auto" || m_dMonitorRules.empty()) {
```
Comment (vaxerski) [inline src/config/ConfigManager.cpp:1654]: ???

Comment (eriedaberrie) [reply src/config/ConfigManager.cpp:1654]: wait what's the issue

Comment (vaxerski) [reply src/config/ConfigManager.cpp:1654]: no {} around short ifs, why did you add them

Comment (eriedaberrie) [reply src/config/ConfigManager.cpp:1654]: I added a multiline else clause so I thought I should add braces around the previous if statements as well, is that not a thing?

Comment (vaxerski) [reply src/config/ConfigManager.cpp:1654]: nope

**Review by vaxerski — APPROVED** [review]
lgtm

**Pull request merged**